pkg/controller/storagecluster/noobaa_system_reconciler.go

```diff
@@ -1,32 +1,33 @@
-package storageclusterinitialization
+package storagecluster
 
 import (
 	"context"
-	"fmt"
-	"os"
 	"reflect"
 
 	"github.com/go-logr/logr"
 	nbv1 "github.com/noobaa/noobaa-operator/v2/pkg/apis/noobaa/v1alpha1"
 	ocsv1 "github.com/openshift/ocs-operator/pkg/apis/ocs/v1"
 	"github.com/openshift/ocs-operator/pkg/controller/defaults"
+	scinit "github.com/openshift/ocs-operator/pkg/controller/storageclusterinitialization"
+	statusutil "github.com/openshift/ocs-operator/pkg/controller/util"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
-func (r *ReconcileStorageClusterInitialization) ensureNoobaaSystem(initialData *ocsv1.StorageClusterInitialization, reqLogger logr.Logger) error {
+func (r *ReconcileStorageCluster) ensureNoobaaSystem(sc *ocsv1.StorageCluster, reqLogger logr.Logger) error {
 
-	nb := r.newNooBaaSystem(initialData, reqLogger)
-	err := controllerutil.SetControllerReference(initialData, nb, r.scheme)
+	nb := r.newNooBaaSystem(sc, reqLogger)
+
+	err := controllerutil.SetControllerReference(sc, nb, r.scheme)
```

Comment:
Can you also add it to the list of relatedObjects?

Reply:
yep, fixed :+1: